scripts/ec2_deploy_teardown/tearDown.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+echo "Obtaining VPC Details"
+VPC=`aws ec2 describe-vpcs --region ${CLUSTER_REGION} --filters "Name=tag:Name,Values=${clusterID}*" --query "Vpcs[0].VpcId" --output text` 
```

Comment:
not required, you don't need VPC for teardown. At least I can't see any usage of it.